### DISCUSSION TITLE
Impossible d'ouvrir le fichier OSF

### DISCUSSION THREAD

**Discussion by tomd1309** [OsmAnd-Rendering/Motorcycle#15] [Q&A] [answered]
Bonjour, je viens de changer de téléphone lorsque j'ai voulu importer le profil il ne passe rien dans l'application (rien à importer). J'ai essayé avec d'autre fichier OSF et ça fonctionne. Pareil si j'exporte un profil et que je le réimporte après ca fonctionne. Mais le fichier Installation.osf ne s'ouvre pas.

**Comment by phpapaziansbd-lab** [comment]
Moi aussi, et pourtant il fonctionnait il y a peu de temps. Que faire ?

**Comment by cricri-du-lauragais** [comment, marked as answer]
je viens d'essayer et sur mon Android 15 avec osmand~ ça fonctionne parfaitement. 
lors de l'ouverture il faut cocher les éléments désirés pour les voir proposé, sinon ça semble vide ;)

**Comment by phpapaziansbd-lab** [comment]
Oui merci, j'ai aussi réussi.

*SBD*
*Philippe Papazian*
*Director Associado*
*www.sbd-repdom.com <http://www.sbd-repdom.com>*
*Cel: 8094245249*
*SECURITY BUSINESS & DOMICILIATION S.R.L*
*Santo Domingo R.D *
